web/src/components/atoms/Input/index.tsx

```diff
@@ -1,6 +1,33 @@
-import { Input, InputProps } from "antd";
+import { Input as AntInput, InputProps } from "antd";
+import { useMemo } from "react";
+import { runes } from "runes2";
 
 export type { SearchProps } from "antd/lib/input";
 
+type Props = {
+  value?: string;
+} & InputProps;
+
+const Input: React.FC<Props> = ({ value, maxLength, ...props }, ref) => {
+  const status = useMemo(() => {
+    if (maxLength && value && runes(value).length > maxLength) {
+      return "error";
+    }
+  }, [maxLength, value]);
+
+  return (
+    <AntInput
```

Comment:
It is better to rename this to a more appropriate name.
Please do the same for all the other components.
```suggestion
    <AntDInput
```